app/src/pages/SingleContent.vue

```diff
@@ -37,32 +37,37 @@ const isExpiredOrScheduled = computed(() => {
     );
 });
 
-watch(
-    content,
-    async () => {
-        if (!content.value) return;
+watch(content, async () => {
+    if (!content.value) return;
 
-        document.title = isExpiredOrScheduled.value
-            ? `Page not found - ${appName}`
-            : `${content.value.title} - ${appName}`;
+    document.title = isExpiredOrScheduled.value
+        ? `Page not found - ${appName}`
+        : `${content.value.title} - ${appName}`;
 
-        if (isExpiredOrScheduled.value) return;
+    // Seo meta tag settings
+    [
+        { name: "title", content: content.value.seoTitle || content.value.title || "" },
```

Comment:
Google does not support the "title" metadata (see https://developers.google.com/search/docs/crawling-indexing/special-tags). We could rather set the document.title to the SEO title if available (in line 43)

Your implementation for the description metadata tag looks good.